Pipfile.lock

```diff
@@ -192,11 +193,11 @@
         },
         "exceptiongroup": {
             "hashes": [
-                "sha256:2ac84b496be68464a2da60da518af3785fff8b7ec0d090a581604bc870bdee41",
-                "sha256:affbabf13fb6e98988c38d9c5650e701569fe3c1de3233cfb61c5f33774690ad"
+                "sha256:542adf9dea4055530d6e1279602fa5cb11dab2395fa650b8674eaec35fc4a828",
+                "sha256:bd14967b79cd9bdb54d97323216f8fdf533e278df937aa2a90089e7d6e06e5ec"
             ],
             "markers": "python_version < '3.11'",
-            "version": "==1.0.0"
+            "version": "==1.0.4"
```

Comment:
The problem we are looking for is there, I think. That's why it doesn't work with python `3.11`, I'm not an expert either on `pipenv` but I'm afraid even if the CI works that it will not work for people using `3.11` no?

Reply:
No, this is saying to only install `exceptiongroup` if the python version is < 3.11. Exception groups are a new feature in 3.11 so this package is back porting it to lower versions.

```json
"exceptiongroup": {
  "hashes": [
    "sha256:542adf9dea4055530d6e1279602fa5cb11dab2395fa650b8674eaec35fc4a828",
    "sha256:bd14967b79cd9bdb54d97323216f8fdf533e278df937aa2a90089e7d6e06e5ec"
   ],
   "markers": "python_version < '3.11'",
   "version": "==1.0.4"
}
```

Reply:
Great! Thank again @sanders41, i will merge your PR 😊